packages/app/src/cli/models/extensions/specifications/app_config_app_proxy.test.ts

```diff
@@ -0,0 +1,52 @@
+import spec from './app_config_app_proxy.js'
+import {describe, expect, test} from 'vitest'
+
+describe('app_config_app_proxy', () => {
+  describe('transform', () => {
```

Comment:
Do you think it's worth to test transform and reverseTransform if they use the default one?